provider/datagen/src/transform/cldr/plurals/mod.rs

```diff
@@ -25,24 +25,24 @@ impl From<&SourceData> for PluralsProvider {
 
 impl PluralsProvider {
     fn get_rules_for(&self, key: ResourceKey) -> Result<&cldr_serde::plurals::Rules, DataError> {
-        match key {
-            CardinalV1Marker::KEY => self
-                .source
+        if key == CardinalV1Marker::KEY {
```

Comment:
Nit: Why did you change this function? I think `match` on ResourceKey should still work (just less efficient)

Reply:
Nope, you can only match on `const` values that have a derived `PartialEq`.

Reply:
This is why I didn't want to add fields to `ResourceKey`

Reply:
I considered that but the if/else is more concise